### PR TITLE
Fabo/app ci fixes

### PR DESCRIPTION
**Pull request by faboweb**

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5edc37ff3846e0d347a23411--lunieio.netlify.app

**Comment by faboweb** [comment]
Working but failing e2e tests don't fail check

File: .github/workflows/app_test.yml [hunk 1]
```diff
@@ -45,5 +45,4 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
       - name: Check if changlog was updated
-        run: npx simsala check --pending-path ./changes
-        working-directory: ${{env.WORKING_DIRECTORY}}
+        run: npx simsala check --pending-path ./app/changes
```
Comment (Bitcoinera) [inline .github/workflows/app_test.yml:48]: Ahh, so this is why it wasn't working? But checked in extension and WORKING_DIRECTORY was right

Comment (faboweb) [reply .github/workflows/app_test.yml:48]: yep, I don't know why it doesn't work with the working directory. But this syntax works.

Comment (Bitcoinera) [reply .github/workflows/app_test.yml:48]: I think it is related with the fact of app always searching for `node_modules` in the root? Like, the `app` folder still "remembers" it was root? 🤔

**Pull request merged**